plots/laws/gravity/throwing_angle.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# Description
+## You throw the grenade to enemies. Which angle of throwing should you choose to hit the farest enemy?
+
+from math import pi
+from sympy import sin, cos
+from sympy.plotting import plot
+from sympy.plotting.plot import MatplotlibBackend
+from symplyphysics import symbols, Eq, pretty, solve, Function
+
+flight_time = symbols('flight_time')
+throwing_velocity = symbols('throwing')
+alpha = symbols('alpha')
+Distance = symbols('Distance')
+velocity_vertical = symbols('v_vertical')
+velocity_horizontal = symbols('v_horizontal')
+velocity_vertical = throwing_velocity * sin(alpha)
+velocity_horizontal = throwing_velocity * cos(alpha)
+time = symbols('time')
+gravitational_constant = symbols('gravitational_constant')
+
+## Height function of time is h = V_vertical * t - g * t**2 /2
+## Flight ends when h == 0
+
+end_of_flight = Eq(0, velocity_vertical * time - gravitational_constant * time**2 /2)
+flight_time = solve(end_of_flight, time, dict=True)[1][time] 
```

Comment:
Move this line of code after the comment. So you can read an explanation before the code does something.